src/main/java/com/minecolonies/coremod/event/ClientEventHandler.java

```diff
@@ -73,6 +83,67 @@ public void renderWorldLastEvent(@NotNull final RenderWorldLastEvent event)
             }
             return;
         }
+        else if (Settings.instance.getBox() != null)
+        {
+            final BlockPos posA = Settings.instance.getBox().getFirst();
+            final BlockPos posB = Settings.instance.getBox().getSecond();
+
+            int x1 = posA.getX();
+            int y1 = posA.getY();
+            int z1 = posA.getZ();
+
+            int x2 = posB.getX();
+            int y2 = posB.getY();
+            int z2 = posB.getZ();
+
+            if (x1 > x2)
+            {
+                x1++;
+            }
+            else
+            {
+                x2++;
+            }
+
+            if (y1 > y2)
+            {
+                y1++;
+            }
+            else
+            {
+                y2++;
+            }
+
+            if (z1 > z2)
+            {
+                z1++;
+            }
+            else
+            {
+                z2++;
+            }
+
+            final double renderPosX = player.lastTickPosX + (player.posX - player.lastTickPosX) * (double)event.getPartialTicks();
+            final double renderPosY = player.lastTickPosY + (player.posY - player.lastTickPosY) * (double)event.getPartialTicks();
+            final double renderPosZ = player.lastTickPosZ + (player.posZ - player.lastTickPosZ) * (double)event.getPartialTicks();
+
+            GlStateManager.enableBlend();
+            GlStateManager.tryBlendFuncSeparate(GlStateManager.SourceFactor.SRC_ALPHA,
+                    GlStateManager.DestFactor.ONE_MINUS_SRC_ALPHA,
+                    GlStateManager.SourceFactor.ONE,
+                    GlStateManager.DestFactor.ZERO);
+            GlStateManager.glLineWidth(2.0F);
+            GlStateManager.disableTexture2D();
+            GlStateManager.depthMask(false);
+
+            final AxisAlignedBB axisalignedbb = new AxisAlignedBB(x1, y1, z1, x2, y2, z2);
+            RenderGlobal.drawSelectionBoundingBox(axisalignedbb.grow(0.002D).offset(-renderPosX, -renderPosY, -renderPosZ), 1.0F, 1.0F, 1.0F, 1.0F);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 0.002D to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)